### PR TITLE
Help Page

### PR DESCRIPTION
**Pull request by yikema**

### CODE REVIEW

File: src/pages/HelpPage.vue [hunk 1]
```diff
@@ -17,22 +19,22 @@
         <card>
           <h2>Attacks Performed On Your Model</h2><hr>
           <p style="line-height:20px">After a thorough literature review of the attack methods of the Cleverhans library, the following attack methods have been chosen to check the robustness of your model.</p>
-          <p style="text-indent:2em">＊ FastGradientMethod</p>
-          <p style="text-indent:2em">＊ BasicIterativeMethod</p>
-          <p style="text-indent:2em">＊ MomentumIterativeMethod</p>
-          <p style="text-indent:2em">＊ CarliniWagnerL2</p>
-          <p style="text-indent:2em">＊ MadryEtAl</p>
+          <p style="text-indent:2em">＊ <a href="https://arxiv.org/pdf/1412.6572.pdf"> FastGradientMethod</a></p>
```
Comment (dnc1994) [inline src/pages/HelpPage.vue]: Maybe delete this list and move the links to the paper titles below.

Comment (yikema) [reply src/pages/HelpPage.vue]: Good idea.

File: src/pages/HelpPage.vue [hunk 2]
```diff
@@ -17,22 +19,22 @@
         <card>
           <h2>Attacks Performed On Your Model</h2><hr>
```
Comment (dnc1994) [inline src/pages/HelpPage.vue]: `<h2>` title here seems too large. Maybe `<h4>` is a better choice. Same for the first title above.

**Review by dnc1994 — COMMENTED** [review]
Looks great. Approved to merge.

**Pull request merged**